tests/acceptance/request_test.py

```diff
@@ -29,8 +31,14 @@ def test_app(request, **overrides):
 def validation_context(request, response=None):
     try:
         yield
-    except Exception:
+    except (
+        exceptions.RequestValidationError,
+        exceptions.ResponseValidationError,
+        exceptions.PathNotFoundError,
+    ):
         raise exception_response(206)
+    except Exception:
+        raise exception_response(400)
 
 
 validation_ctx_path = 'tests.acceptance.request_test.validation_context'
```

Comment:
What test do we need to add so the issue you ran into is covered? (namely that one request flow should produce a RequestValidationError not a SwaggerMappignError)